index.js

```diff
@@ -0,0 +1,21 @@
+const hamburger = document.querySelector('.hamburger');
+const navList = document.querySelector('.nav-list');
+const navLinks = document.querySelectorAll('.nav-list a');
+function toggleBurger() {
+  navList.classList.toggle('nav-active');
+  navLinks.forEach((link, index) => {
+    if (link.style.animation) {
+      link.style.animation = '';
+    } else {
+      link.style.animation = `navLinkFade 0.5s ease forwards ${index / 2 + 0.4}s`;
+    }
+  });
+  hamburger.classList.toggle('toggle');
```

Comment:
- [ ] Great job you did so far but I'm afraid that you might have missed a very small detail that you should consider.
When the user clicks the hamburger button, the menu opens ✔️ , when the user clicks the X button, the menu closes ✔️ , and when the user clicks one of the menu links (Portfolio, About, Contact), the menu closes ✔️ , but the button should doesn't toggle ❌ . Instead, it should get back to the hamburger button!

_Kindly take a look at the two images down below._
### **Your design**
> The button should be the hamburger in this case!
![Screenshot](https://user-images.githubusercontent.com/47625407/122867050-3c141100-d329-11eb-99ea-d00ab287f080.png)

> The button should be the X in this case!
![Screenshot1](https://user-images.githubusercontent.com/47625407/122867087-46360f80-d329-11eb-9a35-f2da56792d5a.png)